packages/components/src/card/compact.tsx

```diff
@@ -0,0 +1,16 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Card from '.';
+import type { TagName, Props } from '.';
+
+const CompactCard = < T extends TagName >( props: Props< T > ): JSX.Element => (
+	<Card { ...props } compact />
+);
```

Comment:
This is actually incorrect as `<CompactCard compact={false} />` would be allowed and we don't want to allow that. However, for whatever reason I cannot for the life of me get `Omit<Props<T>, 'compact'>` to allow me to spread `props` into `Card` and include a `compact` prop afterwards.

Reply:
@sirreal Do you know why `Omit<Props<T>, 'compact'>` isn't working here?

Reply:
I tried to get this working and never figured it out. It doesn't seem like it should be an issue in practice. At most, folks pass an extraneous prop.